build.gradle

```diff
@@ -165,17 +165,18 @@ clean {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:3.0.0')
+    api platform('com.exactpro.th2:bom:3.2.0')
 
-    implementation 'com.exactpro.th2:common:3.18.1'
-    implementation 'com.exactpro.th2:grpc-woodpecker:0.0.2'
+    implementation 'com.exactpro.th2:common:3.39.0'
+    implementation 'com.exactpro.th2:grpc-woodpecker:0.1.0-TH2-3835-+'
 
-    implementation 'org.slf4j:slf4j-log4j12'
-    implementation 'org.slf4j:slf4j-api'
+    implementation 'org.apache.logging.log4j:log4j-slf4j-impl'
+    implementation 'org.apache.logging.log4j:log4j-1.2-api'
+    implementation 'org.apache.logging.log4j:log4j-api'
```

Comment:
We should use slf4j only in projects to provide the ability to switch between implementation in common